docs/workers.md

```diff
@@ -206,6 +206,7 @@ information.
     ^/_matrix/client/(api/v1|r0|v3|unstable)/rooms/.*/members$
     ^/_matrix/client/(api/v1|r0|v3|unstable)/rooms/.*/state$
     ^/_matrix/client/(v1|unstable/org.matrix.msc2946)/rooms/.*/hierarchy$
+    ^/_matrix/client/(v1|unstable/org.matrix.msc2716)/rooms/.*/batch_send$
```

Comment:
This (and other references) are incorrect, it is not available under a `v1` endpoint yet, only the unstable version.